ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,16 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
-    pass
+    self.storage[self.current] = item
 
+    if self.current < self.capacity - 1:
+      self.current += 1
+    else:
+      self.current = 0
+    
   def get(self):
```

Comment:
Great job here, however this could have been done in a more efficient way without creating any extra list and using any extra space, if you would have used a list comprehension to filter our the None values.

Reply:
Good point. Thanks.